draft-ietf-masque-connect-ip.md

```diff
@@ -497,18 +497,18 @@ that support IPv6 MUST ensure that the CONNECT-IP tunnel link MTU is at least
 1280 (i.e., that they can send HTTP Datagrams with payloads of at least 1280
 bytes). This can be accomplished using various techniques:
 
-* if HTTP intermediaries are not in use, both CONNECT-IP endpoints can pad the
-  QUIC INITIAL packets of the underlying QUIC connection that CONNECT-IP is
-  running over.
-
-* if HTTP intermediaries are in use, CONNECT-IP endpoints can enter in an out of
-  band agreement with the intermediaries to ensure that endpoints and
-  intermediaries pad QUIC INITIAL packets.
+* if both endpoints know for certain that HTTP intermediaries are not in use,
+  the endpoints can pad the QUIC INITIAL packets of the underlying QUIC
+  connection that CONNECT-IP is running over to 1321 bytes or more.
```

Comment:
It might be a good idea to show working, because this relies on assumptions.

A maximal QUIC header is 25 bytes (type=1, cid=20, pn=4), assuming that connection IDs in that version of QUIC are limited to 20.  The AEAD takes up to 16, assuming the current set of AEAD functions TLS 1.3 supports.  A datagram frame that consumes the entire payload takes a single byte for the frame identifier, but up to 8 bytes to identify the request stream.  That means you have 25+16+9 bytes of overhead, producing a limit of 1330.

Reply:
Good catch, fixed.